firmware/qemu/src/bin/log.rs

```diff
@@ -419,6 +419,22 @@ fn main() -> ! {
         defmt::info!("true, [1, 2]: {:?}", dhcp_repr);
     }
 
+    defmt::info!(
+        "i128: 0 = {:i128}, -1 = {:i128}, MAX = {:i128}, MIN = {:i128}",
+        0,
+        -1,
+        i128::max_value(),
+        i128::min_value()
+    );
+
+    defmt::info!(
```

Comment:
could you also test here that we can format `u128` and `i128` values using the `{:?}` format specifier?

Reply:
Is it ok now?

Reply:
yes, looks good; thanks! I'll re-approve the PR but it will need a rebase before it can be merged.